app/Config/Bonfire.php

```diff
@@ -9,4 +9,18 @@ class Bonfire extends BaseConfig
     public $views = [
         'filter_list' => 'Bonfire\Views\_filter_list',
     ];
+
+    /**
+     * --------------------------------------------------------------------------
+     * Components' default lookup paths
+     * --------------------------------------------------------------------------
+     *
+     * The list of paths to look for a component as a fallback (by order of priority).
+     * Useful when having a components library to be used across multiple themes.
+     * 
+     * NB: paths MUST end with a slash (/)
+     */
+    public $componentsLookupPaths = [
```

Comment:
This is great, but please add to `app/Config/Themes.php` as it has a components-related setting there already.